### ISSUE TITLE
Africa Regional page showing Global summary

### ISSUE THREAD

**Issue by mmusori** [IFRCGo/go-api#1008]
The Africa Regional page is pulling in Key figures for all emergency operations as opposed to filtering for Africa region only. The other Regional pages seem to be function correctly (Americas and AP at a minimum).  
  
Home page
![image](https://user-images.githubusercontent.com/7780809/107758375-10539f80-6d27-11eb-82ae-77b6ea82f89a.png)

Africa Region Page
![image](https://user-images.githubusercontent.com/7780809/107758389-177aad80-6d27-11eb-9025-41a616845351.png)

Expect behaviour is that the Africa region page should only the number DREFs, Appeal , funding requirements , funding coverage and target population for Africa Region. As is currently displayed on the Americas and Asia Pacific Regional pages. 
 
![image](https://user-images.githubusercontent.com/7780809/107758447-2bbeaa80-6d27-11eb-8493-1945330b9218.png)

**Comment by GregoryHorvath** [comment]
@mmusori The fix is deployed to staging.

@tovari Now we're using the new `/api/v2/appeal/aggregated/` endpoint for the region and country pages. The homepage key-figures would need a bigger refactor for this and we use the Appeals' data there for other things anyway so I kept that as it is for now.

**Comment by mmusori** [comment]
Thanks @GregoryHorvath :
1. There is a slight discrepancy between GO and the donor response. GO has 57.4% coverage vs donor response at 55% .  The Donor response is run only once a day at 9am  so I will check this again in the morning. 
![image](https://user-images.githubusercontent.com/7780809/107784205-73efc400-6d4b-11eb-87b3-16914fa95413.png)
Donor response from IFRC.org
![image](https://user-images.githubusercontent.com/7780809/107784060-4e62ba80-6d4b-11eb-888a-9c2c74ec676f.png)
2. Figures for the Emergency appeal line up with the BO report I downloaded this morning. But the Minor Emergencies do not. Setting the filter to End date after 12/02/21 as discussed earlier (API logic) I get 17 DREFS while the site has 19. The problem is that MDRMZ016 is reflected as an EAP on GO while it is now a DREF, @tovari  can give a full history on this operation. The second problem is that MDRET024 is now part of MDRTIGRAY.   If you are able to quickly resolve these issues before deploying to production that would be great otherwise we deploy and then resolve afterwards.

**Comment by GregoryHorvath** [comment]
@mmusori Keep in mind that on staging we might have additional testing data which could alter these numbers. The only thing I did is basically fix the filtering for the Africa region. Africa has the `ID = 0` and with the previous method of checking `0` was equals to `false` so the code assumed we don't want to filter to any Region so thus why you saw the global figures same as on the home page there.
So if the calculations we just discussed today are correct then we have nothing to fear in this regard.

**Comment by mmusori** [comment]
> @mmusori Keep in mind that on staging we might have additional testing data which could affect these numbers. The only thing I did is basically fix the filtering for the Africa region. Africa has the `ID = 0` and with the previous method of checking `0` was equals to `false` so the code assumed we don't want to filter to any Region so thus why you saw the global figures same as on the home page there.
> So if the calculations we just discussed today are correct then we have nothing to fear in this regard.
> 
> If that's okay I'd say we tackle the second part of your comment separately / after the release, if it still remains an issue.
> 
> Edit: we're calculating EAP/FBA Appeals only into 'Target population' and 'Funding requirements', but not into 'Funding coverage' cc @tovari.

**Comment by mmusori** [comment]
@GregoryHorvath we can push this into production based on your analysis above. I will open a separate ticket on the TIGRAY and EAP issues  since these different issues.

**Comment by batpad** [comment]
Closing this since marked as `Done`.
